build.gradle.kts

```diff
@@ -27,6 +27,7 @@ plugins {
     id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.8.0-RC"
 }
 apiValidation {
+    nonPublicMarkers.add("co.touchlab.kermit.ExperimentalKermitApi")
```

Comment:
This line means annotated items don't contribute to the binary compatibility validator checks. It occurs to me now that there's an argument for not doing this, so that we see breakages even if we don't consider them stable API yet. Thoughts?

Reply:
Let's chat about it today. If the api was in heavy flux I think we'd want to add this. I don't expect it to change much, or at least, I'd only expect it to change a small number of times, so if the api changes need to be reviewed, I don't think that's a huge burden on the dev.

Reply:
Yea i think its probably better to just include it so we can keep track of any api changes for stuff that isn't actively churning a lot